packages/compiler-dom/__tests__/index.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { compile } from '../src'
+
+describe('compile', () => {
+  it('should contain standard transforms', () => {
+    const result = compile(`<div v-text="text"></div>
+        <div v-html="html"></div>
+        <div v-cloak>test</div>
+        <div style="color=red">red</div>
+        <div :style="{color: 'green'}"></div>`)
+
+    expect(result).toMatchSnapshot()
```

Comment:
Just snapshotting the code is fine.

```suggestion
    expect(result.code).toMatchSnapshot()
```